index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Apertium APy</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <h1>Apertium APy</h1>
+    <h3><a href="http://wiki.apertium.org/wiki/Apertium-apy">Documentation</a></h3>
+    <h3><a href="https://github.com/apertium/apertium-apy">Source</a></h3>
+</body>
+</html>
```

Comment:
I think you need to merge origin master into your branch.

Reply:
I have merged origin master now, sorry for this.